### PR TITLE
README: Demonstrate passing subscription id to New-TestResources.ps1

### PR DESCRIPTION
**Pull request by danielmarbach**
When you don't specify the subscription id it takes the default value `faa080af-c1d8-40ad-9cce-e1a450ca5b57` and then you get these errors. Took me a while to spot the default value there :(

```
WARNING: The access token is from the wrong issuer 'https://sts.windows.net/../'. It must match the tenant 'https://sts.windows.net/..47/' associated with this subscription. Please use the authority (URL) 'https://login.windows.net/..47' to get the token. Note, if the subscription is transferred to another tenant there is no impact to the services, but information about new tenant could take time to propagate (up to an hour). If you just transferred your subscription and see this error message, please try back later.
WARNING: Attempt 1 failed: Exception calling "Invoke" with "0" argument(s): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The provided account ..e65 does not have access to subscription ID "faa080af-c1d8-40ad-9cce-e1a450ca5b57". Please try logging in with different credentials or a different subscription ID.". Trying again in 10 seconds...
```

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: eng/common/TestResources/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ is a member of multiple subscriptions.
 
 ```powershell
 Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
```
Comment (jsquire) [inline eng/common/TestResources/README.md:30]: This seems odd to me that we have an internal subscription default here...  

@heaths, @pakrym:  Should we remove the default and force this to be provided?

**Review by jsquire — APPROVED** [review]
Thanks, @danielmarbach.   The change looks good and should definitely help avoid confusion.

**Comment by jsquire** [comment]
@danielmarbach:  There's one gotcha with this change:  the common engineering components are sourced from another repository and then synchronized to the different language repos.   This change needs to be made [here](https://github.com/Azure/azure-sdk-tools/tree/master/eng/common/TestResources).    

I'll leave that to your discretion;  I can replicate this to the tools repo if you'd prefer not to do so.

**Comment by danielmarbach** [comment]
https://github.com/Azure/azure-sdk-tools/pull/1513

File: eng/common/TestResources/README.md (hunk 1 above)
Comment (pakrym) [reply eng/common/TestResources/README.md:30]: it's very convenient for members of our team, maybe we need to check access before defaulting. I don't know if that's possible though.

**Pull request closed without merging**